src/strings/PowerFxResources.en-US.resx

```diff
@@ -3621,6 +3621,17 @@
     <value>https://go.microsoft.com/fwlink/?linkid=2132570</value>
     <comment>{Locked}</comment>
   </data>
+  <data name="ErrorResource_ErrBehaviorFunctionInDataUDF_ShortMessage" xml:space="preserve">
+    <value>Behavior function in a non-behavior user-defined function. Please wrap user-defined function with {} to declare a behavior UDF.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrorResource_ErrBehaviorFunctionInDataUDF_LongMessage" xml:space="preserve">
+    <value>Behavior functions change the state of the app by changing values elsewhere in the app. 'Navigate', 'Patch', 'UpdateContext', and 'Collect' are common behavior functions. 'OnSelect', 'OnVisible', and other 'On …' properties are common behavior-based properties.</value>
```

Comment:
>Navigate [](http://example.com/codeflow?start=100&length=8)

Please add a <comment> node to this resource, and add the directives {Locked=Navigate}{Locked=Patch}... to it. Those function names are not localized, and we don't want them translated by mistake. The comment should also provide additional context to the translation team about how this message is shown to users.